controllers/gslb_controller.go

```diff
@@ -53,7 +53,7 @@ type GslbReconciler struct {
 }
 
 const (
-	gslbFinalizer           = "finalizer.k8gb.absa.oss"
+	gslbFinalizer           = "k8gb.absa.oss/finalizer"
```

Comment:
This might not work well with existing gslbs having "finalizer.k8gb.absa.oss" finalizer set. Imo we need to use an array of valid finalizers here (`["finalizer.k8gb.absa.oss" , "k8gb.absa.oss/finalizer"]`) in order to support smooth upgrade for set/get finalizer logic.

Reply:
@somaritane Conceptually agreed. But how the removal logic will work in that case? Just in case, in order to delete resource we need to remove *all* finalizers from the object

Reply:
Sounds like a plan

Reply:
amended, test coverage for finalizers